flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/sink/SinkOperatorFactory.java

```diff
@@ -44,7 +45,7 @@
         extends AbstractStreamOperatorFactory<byte[]>
         implements OneInputStreamOperatorFactory<InputT, byte[]>, YieldingOperatorFactory<byte[]> {
 
-    private final Sink<InputT, CommT, WriterStateT, ?> sink;
+    @VisibleForTesting public final Sink<InputT, CommT, WriterStateT, ?> sink;
```

Comment:
🙈 Please add a setter for that.

Reply:
I created a getter which is anotated. Is this what you mean?